src/test/java/steps/NavigationResultSteps.java

```diff
@@ -0,0 +1,12 @@
+package steps;
+
+import pages.NewsPage;
+
+public class NavigationResultSteps {
+
+    private NewsPage newsPage = new NewsPage();
```

Comment:
Is it ok that NewsPage is used inside NavigationResultSteps? According to names they are different...
What for 'NavigationResultSteps' is used?

Reply:
NewsPage and others (for example Catalogpage) can also be used here as a result of navigation.  What is the best way to rename this page?